lib/api/types.ts

```diff
@@ -107,3 +107,30 @@ export interface CinemaSearchResponse {
   cinemas: Cinema[]
   pagination: Pagination
 }
+
+// Showtime Types
+export interface Showtime {
+  id: number
+  room_id: number
+  start_time: string // Format: "HH:mm" (e.g., "14:30")
+  end_time: string // Format: "HH:mm" (e.g., "16:45")
+  booked_seats: number
+  room_seats: number
+}
+
+export interface ShowtimeRoom {
+  id: number
+  room_id: number
+  room_name: string
+  showtimes: Showtime[]
+}
```

Comment:
Do we have the Room type? if not rename this to just `Room` so it match the model from the DB